src/components/StudyDetail/DetailBody/style.ts

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+
+export const BodyWrapper = styled.div`
+  width: 600px;
+  margin: 0 auto;
+`;
+
+export const Description = styled.div`
+  padding: 10px 5px;
```

Comment:
피그마에선 본문 내용이 indent가 적용되지 않는 것 같습니다!
```suggestion
  padding: 10px 0;
```

Reply:
해당 부분은 인덴트가 있는것이 좀 더 보기 좋다고 생각해서 넣었습니다~